UI/MainMenu/MainMenu.tscn

```diff
@@ -1,524 +1,551 @@
+columns = 5
+
+[node name="ColorRect" type="Panel" parent="CentrPanel/MarginContainer"]
+modulate = Color( 1, 1, 1, 0.196078 )
+margin_right = 714.0
+mouse_default_cursor_shape = 8
+custom_styles/panel = SubResource( 11 )
+
+[node name="UsefulPlaceholder" type="Control" parent="CentrPanel"]
+margin_top = 695.0
+margin_right = 714.0
 margin_bottom = 720.0
-rect_min_size = Vector2( 5, 0 )
+rect_min_size = Vector2( 0, 25 )
 
-[node name="Panel" type="Panel" parent="HBoxContainer"]
-margin_left = 1040.0
-margin_right = 1280.0
+[node name="RightPanel" type="Panel" parent="."]
+show_behind_parent = true
+margin_left = 1034.0
+margin_right = 1294.0
 margin_bottom = 720.0
-rect_min_size = Vector2( 240, 0 )
-size_flags_horizontal = 3
-size_flags_stretch_ratio = 0.3
-custom_styles/panel = SubResource( 2 )
+rect_min_size = Vector2( 260, 0 )
+custom_styles/panel = SubResource( 3 )
 
-[node name="MarginContainer" type="MarginContainer" parent="HBoxContainer/Panel"]
+[node name="MarginContainer" type="MarginContainer" parent="RightPanel"]
 anchor_right = 1.0
 anchor_bottom = 1.0
-margin_left = 12.0
-margin_top = 12.0
-margin_right = -12.0
-margin_bottom = -12.0
+custom_constants/margin_right = 16
+custom_constants/margin_top = 16
+custom_constants/margin_left = 16
+custom_constants/margin_bottom = 16
+
+[node name="ContentPanel" type="VBoxContainer" parent="RightPanel/MarginContainer"]
+margin_left = 16.0
+margin_top = 16.0
+margin_right = 244.0
+margin_bottom = 704.0
+
+[node name="Scroll" type="ScrollContainer" parent="RightPanel/MarginContainer/ContentPanel"]
+margin_right = 228.0
+margin_bottom = 688.0
 size_flags_horizontal = 3
 size_flags_vertical = 3
+scroll_horizontal_enabled = false
+script = ExtResource( 27 )
 
-[node name="VBoxContainer" type="VBoxContainer" parent="HBoxContainer/Panel/MarginContainer"]
-margin_right = 216.0
-margin_bottom = 696.0
+[node name="ChangebleContent" type="TabContainer" parent="RightPanel/MarginContainer/ContentPanel/Scroll"]
+margin_right = 228.0
+margin_bottom = 688.0
+size_flags_horizontal = 3
+size_flags_vertical = 3
+custom_styles/panel = ExtResource( 4 )
+tabs_visible = false
 
-[node name="GithubReleaseLabel" type="Label" parent="HBoxContainer/Panel/MarginContainer/VBoxContainer"]
-margin_right = 216.0
+[node name="GithubPanel" type="VBoxContainer" parent="RightPanel/MarginContainer/ContentPanel/Scroll/ChangebleContent"]
+anchor_right = 1.0
+anchor_bottom = 1.0
+size_flags_horizontal = 3
+size_flags_vertical = 3
+
+[node name="GithubInfo" type="Label" parent="RightPanel/MarginContainer/ContentPanel/Scroll/ChangebleContent/GithubPanel"]
+margin_right = 228.0
 margin_bottom = 43.0
-text = "Latest github release: loading"
+text = "Latest Github relese:
+loading..."
 autowrap = true
 
-[node name="GithubReleasesButton" type="Button" parent="HBoxContainer/Panel/MarginContainer/VBoxContainer"]
+[node name="ToGihubBt" type="Button" parent="RightPanel/MarginContainer/ContentPanel/Scroll/ChangebleContent/GithubPanel"]
```

Comment:
Old name was better. Just type out Button fully, you won't die

Reply:
Ha-ha-ha)
It's a bit new habit. Some nodes in naming gets too long (and with path in scene much longer!), but my screen too small and long paths hard to review.